### ISSUE TITLE
TestMutationResizeMemoryUp 8.x red cluster health

### ISSUE THREAD

**Issue by pebrc** [elastic/cloud-on-k8s#5023]
Cluster health drops into `red` during this test when running against 8.x snapshots. Example: https://devops-ci.elastic.co/job/cloud-on-k8s-e2e-tests-snapshot-versions-8x/292//testReport:

```
steps_mutation.go:130: Elasticsearch cluster health check failure at 2021-11-08 05:42:08.134813431 +0000 UTC m=+5751.267775538: cluster health red
```

I have not been able to reproduce the issue manually yet. Our e2e test monitoring is currently not working 😞 so we don't have the logs of the Pods before the upgrade. 

It is unclear at this point whether we have found a bug in Elasticsearch or in the orchestration or whether it is something specific to the test environment.

**Comment by pebrc** [comment]
This is #5040

**Comment by thbkrkr** [comment]
Closing as the issue in Elasticsearch responsible for this has been fixed via #5040.
